deployment/scripts/fuel-core-ingress-deploy.sh

```diff
@@ -10,11 +10,21 @@ if [ "${k8s_provider}" == "eks" ]; then
     echo "Updating your kube context locally ...."
     aws eks update-kubeconfig --name ${TF_VAR_eks_cluster_name}
     cd ../ingress/${k8s_provider}
+    echo "Deploying production cluster issuer to ${TF_VAR_eks_cluster_name} ...."
+    mv prod-issuer.yaml prod-issuer.template
+    envsubst < prod-issuer.template > prod-issuer.yaml
+    rm prod-issuer.template
+    kubectl apply -f prod-issuer.yaml
     echo "Deploying fuel-core ingress to ${TF_VAR_eks_cluster_name} ...."
     mv fuel-core-ingress.yaml fuel-core-ingress.template
     envsubst < fuel-core-ingress.template > fuel-core-ingress.yaml
     rm fuel-core-ingress.template
     kubectl apply -f fuel-core-ingress.yaml
+    echo "Deploying monitoring ingress to ${TF_VAR_eks_cluster_name} ...."
```

Comment:
Also not related to fuel-core

Reply:
also now - included in current infrastructure PR #31 on the board- set it up on create-k8s.sh script